### PR TITLE
Ru/importer renaming

### PR DESCRIPTION
**Pull request by rudouglas**

### CODE REVIEW

File: importer-config.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
 # How to find your accountId: https://docs.newrelic.com/docs/accounts/accounts-billing/account-setup/account-id/
-accountId=
+accountId=10956800
 # How to create a User API Key: https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/#user-key-create
-apiKey=
+apiKey=NRII-5DYAYH7YpiNVIThYHXRz75z1Ld8ls_9N
```
Comment (jbeveland27) [inline importer-config.sh:6]: @rudouglas You're gonna want to rotate this API key 😄

File: .github/actions/import-data/src/importer.ts [hunk 2]
```diff
@@ -86,10 +86,10 @@ const createPolicy = async (accountId: number, pack: string) => {
 	return response.alertsPolicyCreate.id;
 };
 
-const createDashboardLocal = async (accountId: number, pack: string) => {
+const createDashboardLocal = async (accountId: number, quickstart: string) => {
 	const replacer = new RegExp('"accountId":0', 'g');
 
-	const dir = `${__dirname}/../../../../packs/${pack}/dashboards`;
+	const dir = `${__dirname}/../../../../quickstarts/${quickstart}/dashboards`;
```
Comment (jbeveland27) [inline .github/actions/import-data/src/importer.ts:92]: One thing that stands out here - quickstarts can now be nested, so this likely doesn't work unless the user realizes `path/to/quickstart` is how the input would need to be structured

**Review by jbeveland27 — APPROVED** [review]
Seems like this should work overall (or at least as much as it "worked" previously) 😄

File: importer-config.sh (hunk 1 above)
Comment (rudouglas) [inline importer-config.sh:6]:
```suggestion
apiKey=
```

Comment (rudouglas) [reply importer-config.sh:6]: says you 😂

File: importer-config.sh [hunk 3]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
 # How to find your accountId: https://docs.newrelic.com/docs/accounts/accounts-billing/account-setup/account-id/
-accountId=
+accountId=10956800
```
Comment (rudouglas) [inline importer-config.sh:4]:
```suggestion
accountId=
```

File: .github/actions/import-data/src/importer.ts (hunk 2 above)
Comment (rudouglas) [reply .github/actions/import-data/src/importer.ts:92]: ya good point, i think we can follow up on this post GA

**Pull request closed without merging**